apex/transformer/testing/standalone_bert.py

```diff
@@ -0,0 +1,221 @@
+import torch
+
+from apex.normalization import FusedLayerNorm as LayerNorm
+from apex.transformer import tensor_parallel
+from apex.transformer.testing.global_vars import get_args
+from .standalone_gpt import get_language_model, get_linear_layer, init_method_normal, parallel_lm_logits, scaled_init_method_normal
+from .standalone_gpt import AttnMaskType, MegatronModule
```

Comment:
apex has enums in https://github.com/NVIDIA/apex/blob/master/apex/transformer/enums.py.
Could you check if the enums.py is au courant with Megatron's def.s and use apex's if possible?